plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSplitSource.java

```diff
@@ -49,6 +49,7 @@
 import static io.trino.plugin.hudi.HudiSessionProperties.getStandardSplitWeightSize;
```

Comment:
please rewrite the commit title to indicate its for hudi

Reply:
DOne

Reply:
This is the commit message guideline: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Done @ebyhr

Reply:
```
    In HUDI connector added exception handling to throw the appropriate exception to prevent queries from hanging incase of an error during split generation

    fixing review comments
```

Please shorten the commit message, e.g. Fix query hang when exception happened in Hudi
Also, remove unhelpful commit body.

Reply:
done